reana_client/cli/quotas.py

```diff
@@ -95,25 +100,33 @@ def quota_show(ctx, access_token, resource, resources, report):  # noqa: D301
             sys.exit(1)
 
         if not report:
-            usage = quota[resource]["usage"]
-            limit = quota[resource]["limit"]
-            health = quota[resource]["health"]
-            percentage = usage_percentage(usage["raw"], limit["raw"])
-            limit_str = (
-                "out of {} used".format(limit["human_readable"])
-                if limit["raw"] > 0
-                else "used"
-            )
+            usage = quota[resource].get("usage")
+            limit = quota[resource].get("limit")
+            limit_str = ""
+            percentage = ""
+            kwargs = dict()
+            if limit:
+                health = quota[resource].get("health")
+                percentage = usage_percentage(usage.get("raw"), limit.get("raw"))
+                if limit > 0:
```

Comment:
Is this check needed? we're already checking `if limit` before.

Reply:
Leftover, thanks.